homeassistant/components/wemo.py

```diff
@@ -76,6 +80,14 @@ def discovery_dispatch(service, discovery_info):
         discovery.load_platform(hass, component, DOMAIN, discovery_info,
                                 config)
 
+    def get_model_from_uuid(uuid):
+        if uuid is None:
+            return None
+        for model, component in WEMO_MODEL_DISPATCH.items():
+            if uuid.startswith('uuid:{}'.format(model)):
+                return model
```

Comment:
Return `component`.

Reply:
Does pywemo not need to know the model name?

Reply:
This function is used to find the component to set up via discovery. That is done in the `discovery_dispatch` function. I don't see any use of the model name.

Reply:
Changing this to `component` results in everything being a switch.